lms/envs/bok_choy.auth.json

```diff
@@ -49,6 +49,15 @@
         ],
         "port": 27017
     },
+    "TRACKING_BACKENDS": {
```

Comment:
This has to match the name of the field that is already defined in aws.py. This just provides an override for BokChoy of the setting which by default just writes to the tracking log.